core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -614,7 +614,7 @@ HIVE_TAR=${HIVE_PREFIX}.tar.gz
 HBASE_MIRROR_URL=http://archive.cloudera.com/cdh5/cdh/5
 HBASE_TAR=hbase-${HBASE_DEP_VER_CDH}.tar.gz
 if [[ "$HBASE_DISTRO" = "HDP" ]]; then
-    HBASE_TAR=hbase-${HBASE_DEP_VER_HDP}.tar.gz
+    HBASE_TAR=hbase-1.1.2.2.3.2.0-2950.tar.gz
```

Comment:
agree with that, but how to decide the version for HBASE_DEP_VER_HDP, if it's a simplified version like 1.1.2, how would a new trafodion user know the actually version. or always use a fix version, like 1.1.2.2.3.2.0-2950?